src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -107,6 +107,9 @@ public Properties getKafkaProducerProperties() {
         producerProps.put("value.serializer", "org.apache.kafka.common.serialization.StringSerializer");
         producerProps.put("key.serializer", "org.apache.kafka.common.serialization.StringSerializer");
         producerProps.put("acks", "all");
+        producerProps.put("request.timeout.ms", kafkaSettings.getRequestTimeoutMs());
```

Comment:
I understand, that it's not in this task, but can you please rename method before from `getKafkaProperties` to `getKafkaConsumerProperties`?